### DISCUSSION TITLE
Add Drupal environment

### DISCUSSION THREAD

**Discussion by bap14** [wardenenv/warden#647] [Ideas]
The nginx container already contains the Drupal configuration, but Warden command doesn't have an environment. https://github.com/wardenenv/warden/pull/646 adds the missing environment for Drupal.

**Comment by navarr** [comment]
Tracking progress on this here: https://github.com/orgs/wardenenv/projects/1?pane=issue&itemId=26226915

Next steps:
* I need to QA your PR and merge
* Documentation should be written on how to setup
